R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -21,10 +21,10 @@ context("SparkSQL functions")
 
 # Utility function for easily checking the values of a StructField
 checkStructField <- function(actual, expectedName, expectedType, expectedNullable) {
-  expect_equal(class(actual), "structField")
-  expect_equal(actual$name(), expectedName)
-  expect_equal(actual$dataType.toString(), expectedType)
-  expect_equal(actual$nullable(), expectedNullable)
+  testthat::expect_equal(class(actual), "structField")
```

Comment:
We don't need to do this -- instead we can include `library(testthat)` in our `lint-R` script as Jenkins and developers who run unit tests should have this package installed.